core/src/mindustry/world/blocks/storage/CoreBlock.java

```diff
@@ -53,7 +53,6 @@ public CoreBlock(String name){
 
         //support everything
         envEnabled = Env.any;
-        drawDisabled = false;
```

Comment:
why was this changed?

Reply:
oh it is duplicated, there is already another one a few lines above